src/test/end-to-end/tests/sorting-list-table-test.ts

```diff
@@ -0,0 +1,102 @@
+import { DateTime } from 'luxon';
+import { createLocation, uploadPublication } from '../shared/testingSupportApi';
+import { randomData } from '../shared/random-data';
+import { config } from '../../config';
+import * as Assert from 'assert';
+
+Feature('Sort List Table');
+
+Scenario('I should be able to view and sort the list table', async ({ I }) => {
+    const displayFrom = DateTime.now().toISO({ includeOffset: false });
+    const displayTo = DateTime.now().plus({ days: 1 }).toISO({ includeOffset: false });
+    const locationId = randomData.getRandomLocationId();
+    const locationName = config.TEST_SUITE_PREFIX + randomData.getRandomString();
+
+    await createLocation(locationId, locationName);
+    await uploadPublication(
+        'PUBLIC',
+        locationId,
+        displayFrom,
+        displayTo,
+        'ENGLISH',
+        'primaryHealthList.json',
+        'PRIMARY_HEALTH_LIST'
+    );
+
+    I.amOnPage('/search');
+    I.see('What court or tribunal are you interested in?');
+    I.fillField('#search-input', locationName);
+    I.click('Continue');
+    I.waitForText('What do you want to view from ' + locationName);
+    I.click(locate('//a').withText('Primary Health Tribunal Hearing List'));
+    I.waitForText('Primary Health\n' + 'Tribunal Hearing List');
+    I.click('Hearing Date');
+    const sortedFirstDateAsc = await I.grabTextFrom('tbody > tr.govuk-table__row:nth-child(1) > td:nth-child(1)');
```

Comment:
The text 'tbody > tr.govuk-table__row:nth-child(1) > td:nth-child({column number})' is used repeated in the test. It is better to define 2 private method firstRow() and lastRow() and pass in the relevant column number. For example:

private void firstRow( private firstRow = (columnNumber): string => {
        return `tbody > tr.govuk-table__row:nth-child(1) > td:nth-child(${columnNumber})`;
    }